### PR TITLE
Cockpit 194

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
preview: https://allisonkarlitskaya.github.io/cockpit-project.github.io/blog/cockpit-194.html

### CODE REVIEW

File: _posts/2019-05-15-cockpit-194.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+category: release
+summary: Firewall page improvements and a new switch
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 194.
+
+### Firewall: Add services to a specific zone
+
```
Comment (martinpitt) [inline _posts/2019-05-15-cockpit-194.md]: We also show the zones on the main firewall page now, right? I think this also deserves a screenshot and explanation.

File: _posts/2019-05-15-cockpit-194.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Services can now be added to and removed from specific zones.
+
+![Adding services to zones](/images/firewall-add-services-to-zones.png)
+
+Furthermore, the firewall page will now give limited feedback in case of
+errors.
```
Comment (martinpitt) [inline _posts/2019-05-15-cockpit-194.md]: This sounds a bit odd -- how about "Dialogs on the Firewall page now show error messages"?

File: _posts/2019-05-15-cockpit-194.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![Firewall error message](/images/firewall-error-message.png)
+
+### Redesigned on/off switch
+
+This release of Cockpit features a redesigned on/off switch.  Compared
+with the old version, the new switch is easier to use (with a bigger
+clickable area) and uses iconography to avoid translation problems.  The
```
Comment (martinpitt) [inline _posts/2019-05-15-cockpit-194.md]: it's also accessible and navigatable with the keyboard now.

File: _posts/2019-05-15-cockpit-194.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+new design is also a better match for PatternFly 4.
```
Comment (martinpitt) [inline _posts/2019-05-15-cockpit-194.md]: This is not that interesting for users, but maybe for developers. How about linking to https://pf4.patternfly.org/components/Switch/examples/ ?

**Comment by garrett** [comment]
Looks good to me!

File: _posts/2019-05-15-cockpit-194.md (hunk 3 above)
Comment (garrett) [reply _posts/2019-05-15-cockpit-194.md]: I guess it's implied, but it's good to point that out directly.

**Pull request merged**